src/net/parsers/texttracks/webvtt.js

```diff
@@ -0,0 +1,311 @@
+/** 
+* Parse WebVTT from text. Returns an array with:
+* - start : start of current cue, in seconds
+* - end : end of current cue, in seconds
+* - content : HTML formatted cue.
+*
+* Global style is parsed and applied to div element. 
+* Specific style is parsed and applied to class element. 
+*
+* @param {string} text
+* @return {Array<Object>}
+*/
+export default function parseWebVTT(text) {
+  const newLineChar = /\n|\s{2,}/g;
+  const linified = text.split(newLineChar);
+  const cuesArray = [];
+  const styleElements = [];
+
+  if (!linified[0].match(/^WEBVTT.*/)) {
+    console.log("Can't parse WebVTT: Non valid file.");
```

Comment:
We don't use the console.log. We either use the ``logger`` in the utils or throw an error.

For the parser, I thinks it's more logical to parse an error.

Reply:
Also why not just ``Invalid file``